mealplanner-ui/src/pages/MealPlans/MealPlans.tsx

```diff
@@ -130,16 +158,45 @@ const MealPlanCard = (props: MealPlanCardProps) => {
               >
                 <ShoppingCart />
               </IconButton>
+
               <IconButton
                 aria-label="delete"
+                onClick = {handleClickOpen}
+              >
+                <DeleteTwoTone />
+              </IconButton>
+              <Dialog
+                fullScreen={fullScreenDialog}
+                open={openDialog}
                 onClick={(e) => {
                   e.stopPropagation();
-                  console.log("meal plan id: ", typeof mealplan.rowId);
-                  deleteMealPlan(connection, mealplan.rowId);
                 }}
+                onClose={handleClose}
+                aria-labelledby="delete-dialog"
               >
-                <DeleteTwoTone />
-              </IconButton>
+                <DialogTitle id="delete-dialog">
+                  {"Delete this meal plan?"}
+                </DialogTitle>
+                <DialogContent>
+                  <DialogContentText>
+                    Are you sure you want to delete this meal plan?
```

Comment:
Please specify the name of the meal plan. Example:
Are you sure you want to **delete** the meal plan _**Vegetarian Only**_ ?